src/remote_debug/remote_debug.cpp

```diff
@@ -1282,14 +1332,22 @@ void remote_debug_start_executable (struct TrapContext *context)
 	uae_u32 ptr = seglist_addr;
 	while(ptr != 0) {
 		char temp[64];
+		unsigned char addrStrTemp[9];
```

Comment:
you can change this loop to have

``addrStrTemp[9] = { 0 };``

Instead and the same for the other variable.